### ISSUE TITLE
[Catalog] No WASM Filters displayed

### ISSUE THREAD

**Issue by leecalcote** [meshery/meshery.io#1304]
### Description
Visit https://meshery.io/catalog/webassembly or click the WebAssembly radio button on meshery.io and find that no WASM filters are displayed.

### Expected Behavior

---
## Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- 📚 See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.meshery.io)

**Comment by nakulbh** [comment]
HEY @leecalcote can I work on this

**Comment by leecalcote** [comment]
@nakulbh, we missed you on this one, but thank you for volunteering!

**Comment by leecalcote** [comment]
@vishalvivekm "wasm filter" is not a category.

**Comment by vishalvivekm** [comment]
> @vishalvivekm "wasm filter" is not a category.

@leecalcote As previously wasm filter was under Technology.
Can I include "wasm filter" in Technology section as a checkbox?

**Comment by Chadha93** [comment]
> > @vishalvivekm "wasm filter" is not a category.
> 
> @leecalcote As previously wasm filter was under Technology. Can I include "wasm filter" in Technology section as a checkbox?

Yes, @vishalvivekm + it should be accessible at https://meshery.io/catalog/webassembly

**Comment by thisiskaransgit** [comment]
@vishalvivekm, while you're on that,  kindly look at the repetition of "deployment" under categories too

**Comment by leecalcote** [comment]
@vishalvivekm @thisiskaransgit please fix today.
